tools/clang/lib/Lex/Pragma.cpp

```diff
@@ -1435,6 +1435,7 @@ void Preprocessor::RegisterBuiltinPragmas() {
     // TODO: add HLSL-specific pragma handlers
```

Comment:
Is this still a todo? Looks like there are a few handlers now

Reply:
Yes, there are more we want to add still.